api_tests/lib_sdk/wallets/index.ts

```diff
@@ -42,16 +50,36 @@ export class Wallets {
         return wallet;
     }
 
-    public async initializeForLedger<K extends keyof AllWallets>(name: K) {
+    public async initializeForLedger<K extends keyof AllWallets>(
+        name: K,
+        logger: Logger,
+        logDir: string,
+        actorConfig: E2ETestActorConfig
+    ) {
         switch (name) {
             case "ethereum":
                 this.wallets.ethereum = new EthereumWallet(
                     global.ledgerConfigs.ethereum
                 );
                 break;
             case "bitcoin":
-                this.wallets.bitcoin = await BitcoinWallet.newInstance(
-                    global.ledgerConfigs.bitcoin
+                if (!this.wallets.bitcoin) {
```

Comment:
Why do we suddenly need this check?

Reply:
Because few lines below we may have already instantiated the bitcoin wallet for lightning purposes.

Reply:
The LN wallet could just instantiate it's own wallet, and this would be gone then :man_shrugging: